docs/get.md

```diff
@@ -0,0 +1,19 @@
+# Get
+
+The get command supports the following sub-commands:
+
+* [Recovery-profile](#recovery-profile)
+
+## Recovery-profile
+
+This commands will display the recovery-profile value set for the osd.
+
+### Example
+
+```bash
+odf get recovery-profile
+```
+
+> ```text
+> Info: high_recovery_ops
```

Comment:
`get` commands should directly print the raw results, without the "Info:" prefix.